wherehows-docs/getting-started.md

```diff
@@ -0,0 +1,141 @@
+    GRANT SELECT ON wherehows.* TO 'wherehows_ro'@'localhost';
+    GRANT SELECT ON wherehows.* TO 'wherehows_ro'@'%';
+
+Run [this sql script](https://github.com/linkedin/WhereHows/blob/master/wherehows-data-model/DDL/create_all_tables_wrapper.sql) to create the required database tables.
+
+
+## Elasticsearch Setup
+WhereHows uses Elasticsearch as its search engine. Please refer to the [Elasticsearch installation guide](https://www.elastic.co/guide/en/elasticsearch/guide/current/running-elasticsearch.html) for setup instructions.
+
+Once you have configured Elasticsearch, run the following command to create `dataset`, `comment` and `field` mappings.
+
+    curl -XPUT '$YOUR_INDEX_URL:9200/wherehows' -d '
+    {
+      "mappings": {
+        "dataset": {},
+        "comment": {
+          "_parent": {
+            "type": "dataset"
+          }
+        },
+        "field": {
+          "_parent": {
+            "type": "dataset"
+          }
+        }
+      }
+    }
+    '
+
+Run the following command to create `flow_job` nested object mappings.
+
+    curl -XPUT '$YOUR_INDEX_URL:9200/wherehows/flow_jobs/_mapping' -d '
+    {
+      "flow_jobs": {
+        "properties": {
+          "jobs": {
+            "type": "nested",
+            "properties": {
+              "job_name":    { "type": "string"  },
+              "job_path": { "type": "string"  },
+              "job_type": { "type": "string"  },
+              "pre_jobs": { "type": "string"  },
+              "post_jobs": { "type": "string"  },
+              "is_current": { "type": "string"  },
+              "is_first": { "type": "string"  },
+              "is_last": { "type": "string"  },
+              "job_type_id": { "type": "short"   },
+          "app_id": { "type": "short"   },
+          "flow_id": { "type": "long"   },
+          "job_id": { "type": "long"   }
+            }
+          }
+        }
+      }
+    }
+    '
+
+Elasticsearch index building is configured as an ETL job in WhereHows. It can be manually triggered by running [ElasticSearchIndex.py](https://github.com/linkedin/WhereHows/blob/master/wherehows-etl/src/main/resources/jython/ElasticSearchIndex.py) script.
+
+
+## Frontend Setup
+WhereHows frontend and backend are independent services and need to be started separately.
+
+Frontend is a standard Ember.js web app running on top of a Play server. Please refer to [Ember](https://www.emberjs.com/) & [Play](https://www.playframework.com/documentation) documents for more details on these frameworks.
+
+### Configuration
+All the frontend-specific configurations are stored in [wherehows-frontend/conf/application.conf](https://github.com/linkedin/WhereHows/blob/master/wherehows-frontend/conf/application.conf). You'll need to change it to match your environment.
+
+### Development
+Run the following command to launch an instance of frontend service locally for development.
+
+    cd wherehows-frontend
+    ../gradlew runPlayBinary
```

Comment:
Yes, but that relies on gradle task name abbreviation: https://docs.gradle.org/3.3/userguide/tutorial_gradle_command_line.html#sec:task_name_abbreviation, which is less clear